### DISCUSSION TITLE
Welcome to odmantic Discussions!

### DISCUSSION THREAD

**Discussion by art049** [art049/odmantic#143] [Announcements]
## 👋 Welcome!
Feel free to ask any questions here and discuss about the project !
Thanks @aahnik for suggesting GH Discussions 😃

**Comment by joeriddles** [comment]


**Reply by aahnik** [reply]
Hi @joeriddles, I don't think the last commit date matters here that much. The project is active as the maintainer is actively looking at issues/PRs/discussions.

odmantic is **fully stable and ready to use in production**. My bot with approx 5K users (as on 26th June, 2021) is powered by odmantic. Read my [experience here ➥](https://github.com/art049/odmantic/discussions/144)

**Comment by chrisdewa** [comment]
Hello! Thanks for ODMantic, i love it from top to bottom. It's perfect for my discord bots and some other side projects. 

Keep rocking.
